devdocs/process.md

```diff
@@ -74,7 +74,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
        git merge master
        ```
 
-7. When the work is ready for review:
+8. When the work is ready for review:
```

Comment:
It is not... I made the change when I thought of having the godmode statement to have a number of its own. Do we create a firstTimer issue for this?